### DISCUSSION TITLE
Old admin console removal

### DISCUSSION THREAD

**Discussion by LoicGudet** [keycloak/keycloak#16158] [Archive]
Hello,

Based on the [Release Plan](https://www.keycloak.org/2022/03/releases.html), the old admin console will be removed.
Do you have a date when this removal will be effective ?

The new one has still some issues and we are not ready to migrate. 
Is there a way to continue using the old admin console even after january ?

Thanks

**Comment by ssilvert** [comment]
We are working toward a final decision on removal and on possibly some way to still run the old console.  No promises, though.

If there are issues with the new console that are showstoppers for you, please open a new issue at https://github.com/keycloak/keycloak-ui/issues and let us know it is critical.

**Reply by kkcmadhu** [reply]
its already removed in keycloak 21 :(
Personal choice,i like the old console much better..
The new one still do not seems to be on par with old one w.r.t to features
e.g i did not find  a way to set the max sessions/tokens allowed per realms,which was recently added.

making changes to policy screen (if you click save), policy get added page does not refresh
like wise adding new role, external identity provider or editing them, no refresh happens, and we really do not know if the changes took effect or not..

To me it looks like the new ui is not production ready yet.. not sure why its removed from keycloak 21.

**Comment by ssilvert** [comment]
@kkcmadhu Thanks for your feedback.  The previous admin console had to be removed as it was very old, hard to maintain, and its framework (Angular 1) is no longer supported.  This also made it a security risk.

The new console should contain all the features of the old one.  Some have been moved or changed slightly.

Can you put your concerns into issues so we can address them?  We generally respond pretty fast.

https://github.com/keycloak/keycloak/issues

**Reply by kkcmadhu** [reply]
> Can you put your concerns into issues so we can address them? We generally respond pretty fast.

Thanks @ssilvert  for quick resoponse,  i have created https://github.com/keycloak/keycloak/discussions/19071

**Comment by ssilvert** [comment]
@kkcmadhu You also mentioned that there was some functionality missing?

Can you provide details in an [issue](https://github.com/keycloak/keycloak/issues)? (not in a discussion)
